lib/api/model/model.dart

```diff
@@ -596,7 +641,8 @@ class ZulipStream {
   ChannelPostPolicy channelPostPolicy;
   // final bool isAnnouncementOnly; // deprecated for `channelPostPolicy`; ignore
 
-  // GroupSettingsValue canRemoveSubscribersGroup; // TODO(#814)
+  GroupSettingValue canAddSubscribersGroup;
+  GroupSettingValue canSubscribeGroup;
```

Comment:
These are documented as new in Zulip 10.0, so we should make them optional.

Reply:
Also we should add these fields to `ChannelUpdateEvent` if they can change with that event, or leave a comment there if they can't.